modules/atomic-opt-in/assets/js/welcome-screen/welcome-popover.js

```diff
@@ -40,16 +40,13 @@ export const WelcomePopover = ( { doClose } ) => {
 	}
 
 	return (
-		<Popover
+		<Dialog
```

Comment:
@hein-obox Please note that the component name should be updated since it's not a popover anymore.